firstvoices/backend/serializers/base_serializers.py

```diff
@@ -134,6 +134,27 @@ class Meta:
         fields = ("visibility",)
 
 
+class UserDetailFieldMixin(metaclass=serializers.SerializerMetaclass):
```

Comment:
This might be more useful as a UserSerializer that could be assigned to different field names